app/assets/images/MergeNB.svg

```diff
@@ -0,0 +1,579 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why change this logo?

Reply:
Describe the method of dynamically changing the logo: if VPD detects the keyword, It will use mount to mount the expected replacement logo to the MergeNB.svg logo.

The  login-controller.html in the branch of OP940 will display two image sources and one Headings1, because we don’t know how to:
1. One of the logo is not displayed.
2. Dynamic modification Headings1.

Therefore, **merge the original two logo and Headings1 into one logo** to realize the above method of dynamically changing the logo.

Reply:
It looks like you took a screenshot of what used to be 2 logos and some text. This won't scale like it used to. 
If you just need to change the IBM logo, why change other areas? 
Could you have an all white logo and just show that when the VPD is OEM? 
Could you just no display the logo when VPD is OEM?

Reply:
> If you just need to change the IBM logo, why change other areas?
> Could you just no display the logo when VPD is OEM?

As I mentioned above, because it is not clear how to make the logo not displayed, and in order to avoid too much difference from the original format, it was modified.

> Could you have an all white logo and just show that when the VPD is OEM?

I also thought about using a white logo to overwrite the existing logo, but as mentioned above, because not sure how to modify Headings1 dynamically, so used the current method.

If you have any suggestions on the above questions, I would be very grateful.

Reply:
Follow the bind mount suggestion but make /usr/share/www/app/assets/images/oem/logo.svg a blank logo 

```
It probably though involves moving the current logo to a new subdir:
/usr/share/www/app/assets/images/logo/
This is because a bind mount replaces the contents of the directory and there are other files in that /usr/share/www/app/assets/images/ directory. Moving the logo will require minor GUI (https://github.com/ibm-openbmc/phosphor-webui/search?q=logo) changes.

Assuming the logo is in this new subdirectory at /usr/share/www/app/assets/images/logo/logo.svg (this is the default logo, so before you bind mount this will be the logo shown on the GUI (this should not be common)).

At build time create 2 new directories:
/usr/share/www/app/assets/images/ibm/
/usr/share/www/app/assets/images/oem/
We think an install -d in the phosphor-webui repo makes the most sense here:
https://github.com/ibm-openbmc/openbmc/blob/OP940/meta-phosphor/recipes-phosphor/webui/phosphor-webui_git.bb#L43

Place their respective logos in the new directories. Call them the same logo.svg. You now have:
/usr/share/www/app/assets/images/logo/logo.svg
/usr/share/www/app/assets/images/ibm/logo.svg
/usr/share/www/app/assets/images/oem/logo.svg <- make this a blank logo 

Now create 2 new systemd units:
"ibm-image.mount" and "oem-image.mount"

These systemd units just bind mount /usr/share/www/app/assets/images/logo/ to either /usr/share/www/app/assets/images/ibm/ or /usr/share/www/app/assets/images/oem/

Some links to help
https://utcc.utoronto.ca/~cks/space/blog/linux/SystemdBindMountUnits
https://www.freedesktop.org/software/systemd/man/systemd.mount.html

The application to start the systemd units will be either openpower-vpd-parser (option 1 we talked about this morning) or a new application that is monitoring for changes to this vpd keyword (option 2 as we talked about this morning). So instead of creating a link as we talked about, you would instead start a systemd unit depending on the value of the new vpd keyword.
This systemd unit would do a bind mount.

After the systemd unit bind mounts, the logo will now come from that directory.

Some feasibility:
# pwd
/var/persist
# mkdir -p www/images/logo
# mkdir -p www/images/ibm
# mkdir -p www/images/oem
# echo 'ibm' > www/images/ibm/logo.svg
# echo 'oem' > www/images/oem/logo.svg
# ls -l www/images/logo
#
# mount --bind www/images/ibm www/images/logo
# ls www/images/logo/
logo.svg
# cat www/images/logo/logo.svg
ibm
# umount www/images/logo
# mount --bind www/images/oem www/images/logo
# cat www/images/logo/logo.svg      
oem
```

Reply:
### - Use the original format in our method and  use blank logo to replace
(keywords exist in the VPD)

![image](https://user-images.githubusercontent.com/63627630/125886688-3e119c8c-83ad-4fff-bb90-b5cfc3d0076c.png)

The key question is, how to replace the content of the Headings1? 
`<h1 class="login__desc">BMC System Management</h1>`
Regardless of the method you provided before, or our current method, it seems that there is no way to achieve it.

Make corrections according to the format provided before (pictured below), that's why I have this question.
![image](https://user-images.githubusercontent.com/63627630/125887463-45464bad-76e7-4665-906d-dac8f32fa11d.png)